image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -121,17 +123,28 @@ object FileMetadataReader {
       }
     }
 
-  def getColorModelInformation(metadata: Metadata): Map[String, String] = {
+  def getColorModelInformation(image: File, metadata: Metadata): Future[Map[String, String]] = {
+
+    val source = addImage(image)
+
+    val formatter = format(source)("%A")
+
+    for {
+      output <- runIdentifyCmd(formatter)
```

Comment:
Doing this makes extracting all the non `hasAlpha` attributes dependent on `runIdentifyCmd` succeeding. It might be better to allow it fail gracefully and still return valuable information.